docs/tutorials/mask_maps.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Is "_maskded" a typo, maybe just "_mask"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
`high_significance_maskded` : it's not a typo I just wanted to stress that high significance are false in this mask

Reply:
<p>I see, maybe there is more explicit suffix? E.g. <code>mask_inv</code></p><p> </p>

Reply:
ok renamed to significance_mask and significance_mask_inv